data/rooms.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import jwt
+from sqlalchemy import Column, Integer, String, DateTime, Boolean
+from data.db import Base
+from server import app, bcrypt, BCRYPT_LOG_ROUNDS
+from sqlalchemy.sql.sqltypes import BOOLEAN, CHAR
+
+class Rooms(Base):
```

Comment:
This class (and db table) is not useful in my opinion because we are not booking multiple rooms in a single reservation. Please let me know your thoughts. I've already added `meditation_room_id` in `reservation` table